From: siretart <siretart@9553f0bf-9b14-0410-a0b8-cfaf0461ba5b>
Date: Mon, 13 Sep 2010 13:25:51 +0000 (+0000)
Subject: fix x11grab example in e.g. the manpage so that they actually work
X-Git-Url: http://git.ffmpeg.org/?p=ffmpeg;a=commitdiff_plain;h=ead47de1d89067e24bc3f520b6d762ecf64db002

fix x11grab example in e.g. the manpage so that they actually work

git-svn-id: file:///var/local/repositories/ffmpeg/trunk@25113 9553f0bf-9b14-0410-a0b8-cfaf0461ba5b
---

diff --git a/doc/ffmpeg-doc.texi b/doc/ffmpeg-doc.texi
index 71bed9b..d9c5c14 100644
--- a/doc/ffmpeg-doc.texi
+++ b/doc/ffmpeg-doc.texi
@@ -817,14 +817,14 @@ standard mixer.
 FFmpeg can grab the X11 display.
 
 @example
-ffmpeg -f x11grab -s cif -i :0.0 /tmp/out.mpg
+ffmpeg -f x11grab -s cif -r 25 -i :0.0 /tmp/out.mpg
 @end example
 
 0.0 is display.screen number of your X11 server, same as
 the DISPLAY environment variable.
 
 @example
-ffmpeg -f x11grab -s cif -i :0.0+10,20 /tmp/out.mpg
+ffmpeg -f x11grab -s cif -r 25 -i :0.0+10,20 /tmp/out.mpg
 @end example
 
 0.0 is display.screen number of your X11 server, same as the DISPLAY environment
